app/ide-desktop/lib/dashboard/src/services/Backend.ts

```diff
@@ -1321,11 +1321,11 @@ export default abstract class Backend {
   /** Return file details. */
   abstract getFileDetails(fileId: FileId, title: string): Promise<FileDetails>
   /** Create a Data Link. */
-  abstract createConnector(body: CreateConnectorRequestBody): Promise<ConnectorInfo>
+  abstract createDataLink(body: CreateDataLinkRequestBody): Promise<DataLinkInfo>
   /** Return a Data Link. */
-  abstract getConnector(connectorId: ConnectorId, title: string | null): Promise<Connector>
+  abstract getDataLink(dataLinkId: DataLinkId, title: string | null): Promise<DataLink>
```

Comment:
all occurrences of `dataLink` or `dataLinkId` should be renamed to `datalink` or `datalinkId`